src/webgpu/shader/execution/expression/call/builtin/abs.spec.ts

```diff
@@ -21,7 +21,7 @@ import { builtin } from './builtin.js';
 
 export const g = makeTestGroup(GPUTest);
 
-g.test('integer_builtin_functions,abs_unsigned')
+g.test('unsigned')
```

Comment:
I'm thinking it might be better to call this `u32` and the signed `i32`. We're likely to support smaller datatypes in the future.

Reply:
Done